### ISSUE TITLE
Some VST3s not supported

### ISSUE THREAD

**Issue by okhick** [ramirezd42/vst-js#11]
I've been trying to use a VST3 exported from Cycling 74's Max MSP Gen~ system. While this plugin works fine in Reaper, I've been having some trouble getting it to work in VST-JS. The pluginHost is launched and a even a small GUI window opens. The audio process starts but the outputBuffer is all zeros. 

Deactivating the `pluginHost.processAudioBlock()` and running the program passes audio just fine so I know it's reading the file and it has something to do with the pluginHost. I know this is a very specific edge case so it's not top priority, but any ideas you have would be welcome!

**Comment by backlit-studio** [comment]
Honestly I would consider any use of this library to be a pretty big edge
case!

I'm away for the week right now but I'd be happy to take a look when I get
back. Feel free to open a PR if you find anything before I do.

Also, have you been able to get ANY plugins to work? I've had success with
voxengos eq. So it's worth a sanity check there I think

http://www.voxengo.com/product/curveeq/

**Comment by okhick** [comment]
Yes other VST3s do work! Sorry for failing to mention that. I'll do some poking and see if I stumble across anything. Let me know if you'd like one of the problem plugins or the Juce project that that goes with it.

This is a really neat project, by the way. Thanks for all the work!
